translations/de/info/task_description.html

```diff
@@ -343,40 +335,40 @@
 </table>
 
 <p>
-    You should help Stephan write a function to calculate this "multiplication".
-    You are given two positive integers (n&nbsp;>&nbsp;0), be careful with order of arguments.
+    Du sollst Stephan helfen eine Funktion zu schreiben, die diese Art von "Multiplikation"
+    anwenden kann.
```

Comment:
we say eine Funktion _führt aus_,  rather than _wendet an_, dh: _..die diese Art von  "Multiplikation" ausführen kann_